<ui version="4.0" >
 <class>SettingsDialogClass</class>
 <widget class="QDialog" name="SettingsDialogClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>560</width>
    <height>517</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Настройки latticePlugin</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" >
   <item row="19" column="1" colspan="3" >
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="17" column="2" >
    <spacer name="verticalSpacer" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="11" column="1" >
    <widget class="QLabel" name="label_2" >
     <property name="text" >
      <string>Директория файлов для обработки:</string>
     </property>
    </widget>
   </item>
   <item row="11" column="2" >
    <widget class="QLineEdit" name="lineEdit_processingDir" />
   </item>
   <item row="11" column="3" >
    <widget class="QToolButton" name="toolButton_setProcessingDir" >
     <property name="text" >
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="12" column="1" >
    <widget class="QLabel" name="label_4" >
     <property name="text" >
      <string>Директория файлов отчётов:</string>
     </property>
    </widget>
   </item>
   <item row="12" column="2" >
    <widget class="QLineEdit" name="lineEdit_reportsDir" />
   </item>
   <item row="12" column="3" >
    <widget class="QToolButton" name="toolButton_setReportsDir" >
     <property name="text" >
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="13" column="1" >
    <widget class="QLabel" name="label_6" >
     <property name="text" >
      <string>Максимальное количество потоков:</string>
     </property>
    </widget>
   </item>
   <item row="13" column="2" >
    <layout class="QHBoxLayout" name="horizontalLayout" >
     <item>
      <widget class="QSpinBox" name="spinBox_maxBPThreadCount" >
       <property name="minimum" >
        <number>1</number>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="16" column="1" colspan="3" >
    <widget class="QCheckBox" name="checkBox_useExternalProcessing" >
     <property name="text" >
      <string>Использовать внешний поток для рассчёта</string>
     </property>
    </widget>
   </item>
   <item row="10" column="1" colspan="3" >
    <widget class="Line" name="line" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="6" column="1" colspan="3" >
    <layout class="QHBoxLayout" name="horizontalLayout_3" >
     <item>
      <widget class="QCheckBox" name="checkBox_showBest" >
       <property name="text" >
        <string>Показать только лучшие</string>
       </property>
       <property name="checked" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBox_Start_Threshold" >
       <property name="toolTip" >
        <string>Порог начала</string>
       </property>
       <property name="maximum" >
        <number>1000000000</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBox_End_Treshold" >
       <property name="toolTip" >
        <string>Порог конца</string>
       </property>
       <property name="maximum" >
        <number>1000000000</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="2" colspan="2" >
    <widget class="QLineEdit" name="lineEdit_programmFileName" />
   </item>
   <item row="3" column="1" >
    <widget class="QLabel" name="label_3" >
     <property name="text" >
      <string>Исполняемый файл:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="3" >
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>Настройки параметров коммандной строки:</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout" >
      <item>
       <widget class="QTableWidget" name="tableWidget_parameters" >
        <column>
         <property name="text" >
          <string>Имя параметра</string>
         </property>
        </column>
        <column>
         <property name="text" >
          <string>Значение</string>
         </property>
        </column>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2" >
        <item>
         <spacer name="horizontalSpacer_2" >
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_addParam" >
          <property name="toolTip" >
           <string>Добавить параметр
</string>
          </property>
          <property name="text" >
           <string>Добавить</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_delParam" >
          <property name="toolTip" >
           <string>Удалить параметр</string>
          </property>
          <property name="text" >
           <string>Удалить</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="1" >
    <widget class="QLabel" name="label_7" >
     <property name="text" >
      <string>Файл латтиса:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2" colspan="2" >
    <widget class="QLineEdit" name="lineEdit_latticeFile" />
   </item>
   <item row="15" column="1" colspan="3" >
    <widget class="Line" name="line_2" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="14" column="1" >
    <widget class="QCheckBox" name="checkBox_usePostProcessing" >
     <property name="text" >
      <string>Использовать пост процессинг</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SettingsDialogClass</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>257</x>
     <y>501</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SettingsDialogClass</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>325</x>
     <y>501</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_showBest</sender>
   <signal>toggled(bool)</signal>
   <receiver>spinBox_Start_Threshold</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>115</x>
     <y>256</y>
    </hint>
    <hint type="destinationlabel" >
     <x>233</x>
     <y>256</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_showBest</sender>
   <signal>toggled(bool)</signal>
   <receiver>spinBox_End_Treshold</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>51</x>
     <y>252</y>
    </hint>
    <hint type="destinationlabel" >
     <x>419</x>
     <y>253</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
